docs/first-steps/security-and-shared-accounts.md

```diff
@@ -17,7 +17,7 @@ cd security
 Move into the `us-east-1/base-tf-backend` directory and run:
 
 ``` bash
-leverage terraform init
+leverage terraform init --skip-validation
 leverage terraform apply
 ```
 
```

Comment:
Down on line [#88](https://github.com/binbashar/le-ref-architecture-doc/blob/4ac88684f87e975efce36f0b57b61d08fc4dc8d8/docs/first-steps/security-and-shared-accounts.md?plain=1#L88), there's a `--skip-validation` flag still missing for the Shared account's backend layer deployment.
Just that little change and this PR seems ready to be shipped!
Thanks in advance @juanmatias !